### ISSUE TITLE
Please add `delta` and `now` functions to make more advanced time-based games

### ISSUE THREAD

**Issue by justgook** [evancz/elm-playground#17]
now there is only `spin`, `wave`, and `zigzag`, but if you would like to make some more advanced games, add physics, animations, particles, you need to know `delta` from previous frame..

`now`  is needed when you want to create some schedule events, and now we cannot do that as we don't know time what is now, and do that based on counting frames not really good way..

**Comment by Snaker1** [comment]
I agree. I just wanted to try this package but had to give up because there is no way to get the delta. I hope this can be implemented soon as this package seems pretty great for learning how to use Elm.

**Comment by mahidul-islam** [comment]
> I agree. I just wanted to try this package but had to give up because there is no way to get the delta. I hope this can be implemented soon as this package seems pretty great for learning how to use Elm.

Hey @Snaker1 I think you can now use this package https://package.elm-lang.org/packages/justgook/webgl-playground/4.1.3/  to have this feature. and start building advance stuff...
